SPECS-DEV/aarch64-mariner-linux-gnu-prebuilt-toolchain/aarch64-mariner-linux-gnu-prebuilt-toolchain.spec

```diff
@@ -0,0 +1,31 @@
+Summary:        Prebuilt aarch64-mariner-linux-gnu cross compiling toolchain
+Name:           aarch64-mariner-linux-gnu-prebuilt-toolchain
+Version:        0.1.0
+Release:        1%{?dist}
+License:        GPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+AutoReqProv:    no
+ExclusiveArch:  x86_64
+Source0:        %{name}-%{version}.tar.gz
```

Comment:
Where is this source coming from? Is it built through a separate step elsewhere similar to how building packages inside `SPECS-SIGNED` work?